Overview.src.html

```diff
@@ -2607,9 +2644,24 @@ <h5 id="calculating-the-iteration-time">Calculating the iteration time</h5>
 1.  If the <a>scaled active time</a> is <a>unresolved</a>,
     return <a>unresolved</a>.
 
-2.  If the <a>iteration duration</a> is zero, return zero.
+2.  If the <a>iteration duration</a> is zero or infinity, return zero.
+
+3.  If the <a>start delay</a> is negative infinity, return the result of
+    evaluating<!-- @ifdef INCLUDE_ANIMATION_NODE_PLAYBACKRATE -->
+    <code>(<a>animation effect playback rate</a> &times; <a>local time</a> +
+    (<a>iteration start</a> % 1) &times; <a>iteration duration</a>) %
+    <a>iteration duration</a></code>.
+
+<!-- @endif --><!-- @ifndef INCLUDE_ANIMATION_NODE_PLAYBACKRATE -->
+    <code>(<a>local time</a> + (<a>iteration start</a> % 1) &times;
+    <a>iteration duration</a>) % <a>iteration duration</a></code>.
+
+<!-- @endif -->
+
+    Note: In this case the <a>iteration count</a> must be positive infinity or
```

Comment:
Don't use MUST here; even though it's in a note and thus by definition not normative, it's still confusing.  Rephrase as "in this case the iteration count is guaranteed to be positive infinity, as otherwise the scaled active time would be unresolved".